### DISCUSSION TITLE
AMC question?

### DISCUSSION THREAD

**Discussion by Karthick2299** [amzn/ads-advanced-tools-docs#171] [Q&A] [answered]
Could you please provide a brief explanation about a campaign and AMC (Amazon Marketing Cloud)? Additionally, I have another question: Can I access past-year advertised product report data using AMC now? 
what is the maximum look back window for AMC in Amazon Ads API ?
Thanks!

**Comment by elizafarley** [comment, marked as answer]
Hi @Karthick2299, AMC is a tool to be used in combination with Amazon DSP and sponsored ads campaigns for reporting, signal management, and advertiser audience management. Note that you **must** be using Amazon DSP to use AMC. See the [getting started guide](https://advertising.amazon.com/API/docs/en-us/guides/amazon-marketing-cloud/get-started/get-started) for prerequisites and more details.

For sponsored ads campaigns, you can see the data available in AMC and how it maps to API reports [here](https://advertising.amazon.com/API/docs/en-us/guides/amazon-marketing-cloud/datasources/sponsored-ads-datasets).

When you start using AMC, you create an instance which initiates with 7 days worth of data, and accumulates up to 12.5 months worth of data. You would not have access to 12.5 months immediately but data would build up after creating you instance. You can see more info on this [here](https://advertising.amazon.com/API/docs/en-us/guides/amazon-marketing-cloud/how-amc-works#amc-instance).
